llama_hub/llama_datasets/paul_graham_essay/README.md

```diff
@@ -0,0 +1,39 @@
+# Paul Graham Essay
+
+A `LlamaDataset` based on the Paul Graham Essay document.
+
+## CLI Usage
+
+You can download `llamadatasets` directly using `llamaindex-cli`, which comes installed with the `llama-index` python package:
+
+```bash
+llamaindex-cli download-llamadataset PaulGrahamEssayDataset --download-dir ./data
```

Comment:
Maybe I missed it, but I didn't see the cli changes yet in the framework PR (that's ok for now, just a heads up)

Reply:
(for transparency)

This has since been added to cli -- thanks Logan!